dcicutils/misc_utils.py

```diff
@@ -152,3 +156,20 @@ def get_setting_from_context(settings, ini_var, env_var=None, default=None):
         if env_var in os.environ:
             return os.environ.get(env_var)
     return settings.get(ini_var, default)
+
+
+# This should move to dcicutils.misc_utils for better sharing
```

Comment:
This comment is now out of date 😄

Reply:
This comment about this being out of date is now out of date. :)